lib/drowsy/relation.rb

```diff
@@ -73,6 +73,18 @@ def to_http_request
     Drowsy::HttpRequest.new(connection, http_method, uri_template, params)
   end
 
+  def method_missing(name, *args, &block)
+    if klass.has_scope?(name)
+      instance_exec(*args, &klass.scope_for(name))
+    else
+      super
+    end
+  end
+
+  def respond_to?(name, include_private = false)
```

Comment:
I think this should be [respond_to_missing?](http://blog.marc-andre.ca/2010/11/15/methodmissing-politely/)